client/my-sites/themes/themes-magic-search-card/index.jsx

```diff
@@ -296,7 +296,7 @@ class ThemesMagicSearchCard extends React.Component {
 							</div>
 						}
 						{
-							( isPremiumThemesEnabled && ( config.isEnabled( 'jetpack/pijp' ) || ! isJetpack ) ) &&
+							isPremiumThemesEnabled &&
```

Comment:
Uh, never mind, I hadn't pulled the latest commit :man_facepalming: Sorry for the noise!